README.md

```diff
@@ -35,6 +35,28 @@ For any other action a warning entry is added to log and Record is skipped.
 | `bulkSize`               | The number of items stored in bulk in the index. The minimum value is `1`, maximum value is `10000`. Note that values greater than `1000` may require additional service configuration.                                                          | `true`                                               | `"1000"` |
 | `retries`                | The maximum number of retries of failed operations. The minimum value is `0` which disabled retry logic. The maximum value is `255`. Note that the higher value, the longer it may take to process retries, as a result, ingest next operations. | `true`                                               | `"1000"` |
 
+
+# Source
+ElasticSearch source connector allows you to move data from multiple Elasticsearch indexes with the specified `host` and `indexes`. It uses elasticsearch search api to pull data from indexes. Upon starting it pulls batches of data from indexes, once all the data is retrieved, it then polls the search api to pull data at regular intervals. 
+
+## Configuration Options
+| name                     | description                                                                                                                                                                                                                                      | required                                             | default  |
+|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|----------|
+| `version`                | The version of the Elasticsearch service. One of: `5`, `6`, `7`, `8`.                                                                                                                                                                            | `true`                                               |          |
```

Comment:
It's only version 8 that is now supported, right?

Reply:
Yes, once this is merged will add support for other versions also.

Reply:
Gotcha... In that case, we should only have 8 mentioned (with a note that we're working on support for others).